hello-kedro/hello-kedro.py

```diff
@@ -0,0 +1,35 @@
+from kedro.io import DataCatalog, MemoryDataSet
+from kedro.pipeline import node, Pipeline
+from kedro.runner import SequentialRunner
+
+# Prepare a data catalog
+data_catalog = DataCatalog({"example_data": MemoryDataSet()})
+
+
+def return_greeting():
+    # Prepare first node
+    return "Hello"
+
+
+return_greeting_node = node(
+    return_greeting, inputs=None, outputs="my_salutation"
+)
```

Comment:
```suggestion
return_greeting_node = node(return_greeting, inputs=None, outputs="my_salutation")
```

Reply:
I ran black formatter.

Reply:
I ran black too with length of 79 char. That line has 82, I think.